tests/generate-data/src/typescript.ts

```diff
@@ -0,0 +1,3 @@
+// TypeScript generates source map that includes mapping for line ending characters
+console.log('hello\n\r');
```

Comment:
Is the EOL here meant to be `\n\r`? If so, `git` has eaten them.

```shell
$ curl https://raw.githubusercontent.com/danvk/source-map-explorer/254a5d997eafb0234385ab52c7ff60714d167a52/tests/generate-data/src/typescript.ts | hexdump -C
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   134  100   134    0     0    206      0 --:--:-- --:--:-- --:--:--   206
00000000  2f 2f 20 54 79 70 65 53  63 72 69 70 74 20 67 65  |// TypeScript ge|
00000010  6e 65 72 61 74 65 73 20  73 6f 75 72 63 65 20 6d  |nerates source m|
00000020  61 70 20 74 68 61 74 20  69 6e 63 6c 75 64 65 73  |ap that includes|
00000030  20 6d 61 70 70 69 6e 67  20 66 6f 72 20 6c 69 6e  | mapping for lin|
00000040  65 20 65 6e 64 69 6e 67  20 63 68 61 72 61 63 74  |e ending charact|
00000050  65 72 73 0a 63 6f 6e 73  6f 6c 65 2e 6c 6f 67 28  |ers.console.log(|
00000060  27 68 65 6c 6c 6f 5c 6e  5c 72 27 29 3b 0a 63 6f  |'hello\n\r');.co|
00000070  6e 73 6f 6c 65 2e 6c 6f  67 28 27 77 6f 72 6c 64  |nsole.log('world|
00000080  5c 6e 27 29 3b 0a                                 |\n');.|
00000086
```

Reply:
Strange. I think git only suppose to replace all EOL to be `\r`. But shouldn't really matter

Reply:
I piped `curl` to `hexdump`, and `.` is `0x0a` (`\n`).  GitHub renders those files line by line, so it will always look fine in browser.

My point is, if this is the input file for testing the implementation in the future, it doesn’t really cover the windows EOL case.

Reply:
> My point is, if this is the input file for testing the implementation in the future, it doesn’t really cover the windows EOL case.

We don't need exactly windows  EOL to reproduce the issue. EOL type doesn't really matter, does it?